src/components/common/Extensions/NotificationBanner/NotificationBanner.tsx

```diff
@@ -3,31 +3,29 @@ import { useIntl } from 'react-intl';
 import clsx from 'clsx';
 
 import Link from '~v5/shared/Link';
-import Icon from '~shared/Icon';
 
 import CopyUrl from './CopyUrl';
 import styles from './NotificationBanner.module.css';
 import { NotificationBannerProps } from './types';
+import StatusText from '~v5/shared/StatusText';
 
 const displayName = 'common.Extensions.NotificationBanner';
 
 const NotificationBanner: FC<PropsWithChildren<NotificationBannerProps>> = ({
   status,
   title = '',
   children,
-  actionText,
-  actionType,
+  action,
   isAlt = false,
-  redirectUrl,
-  onClick,
 }) => {
   const { formatMessage } = useIntl();
   const titleText = typeof title === 'string' ? title : formatMessage(title);
   const actionMessage =
-    typeof actionText === 'string'
-      ? actionText
-      : actionText && formatMessage(actionText);
-  // @TODO: handle actionType 'call-to-action'
+    action &&
+    (typeof action.actionText === 'string'
```

Comment:
Maybe you can take `actionText` out of `action` earlier and then use it as `actionText` instead of `action.actionText`?

Reply:
Fixed